mithril-client/src/services/snapshot.rs

```diff
@@ -237,16 +238,30 @@ impl SnapshotService for MithrilClientSnapshotService {
             .unwrap()
             .with_key("eta", |state: &ProgressState, w: &mut dyn Write| write!(w, "{:.1}s", state.eta().as_secs_f64()).unwrap())
             .progress_chars("#>-"));
-        let filepath = self
+        let snapshot_path = self
             .snapshot_client
-            .download(&snapshot_entity.artifact, pathdir, pb)
+            .download(&snapshot_entity.artifact, download_dir, pb)
             .await
-            .map_err(|e| format!("Could not download file in '{}': {e}", pathdir.display()))?;
+            .map_err(|e| {
+                format!(
+                    "Could not download file in '{}': {e}",
+                    download_dir.display()
+                )
+            })?;
 
         progress_bar.println("5/7 - Unpacking the snapshot…")?;
-        let unpacker = unpacker.unpack_snapshot(&filepath, &unpack_dir);
+        let unpacker = unpacker.unpack_snapshot(&snapshot_path, &unpack_dir);
         self.wait_spinner(&progress_bar, unpacker).await?;
 
+        // Append 'clean' file to speedup node bootstrap
+        if let Err(error) = File::create(unpack_dir.join("clean")) {
+            warn!(
+                "Error while creating of a 'clean' file in the unpacked directory to speed\
```

Comment:
typo: remove `of a`

Reply:
suggestion: if it's a `warn!` then do not use `Error` in the message or someone looking at the logs might get confused. Rewrite message as something like:
> Could not create clean shutdown marker file in directory {dir}: {err}